index.php

```diff
@@ -2347,6 +2347,22 @@ function ProcessChildrenOf($boxparent, &$sql, &$numthispage, &$secondcol, &$thir
 		}
 		$dvd['format'] .= ' ' . $colours;
 
+		if ($dvd['format'] != '') {
```

Comment:
This block is useless. We know for a fact that `$dvd['format']` is not empty (it has at least a space, due to line 2348).